Classes/Service/Tika/AppService.php

```diff
@@ -61,7 +62,7 @@ protected function initializeService() {
 	public function getTikaVersion() {
 		$tikaCommand = CommandUtility::getCommand('java')
 			. ' -Dfile.encoding=UTF8' // forces UTF8 output
-			. ' -jar ' . escapeshellarg(GeneralUtility::getFileAbsFileName($this->configuration['tikaPath'], FALSE))
+			. ' -jar ' . ShellUtility::escapeShellArgument(GeneralUtility::getFileAbsFileName($this->configuration['tikaPath'], FALSE))
```

Comment:
I don't think we need to escape the path to the Tika jar. It's probably more likely that a non-ASCII character is being used in the file name we're trying to pass to Tika.

This would then apply to the other changes as well.